R/r_error.R

```diff
@@ -10,6 +10,9 @@ r_error <- function(error = TRUE) {
   # Getting the session of the student
   environment <- environment_info(error)
 
+  thinks <- paste("RAINER analyzes your error message: \n", environment$error_message)
+  cat(thinks)
```

Comment:
You can just call cat(paste()) here I'd say (or even skip the paste, as cat already concatenates strings)